### DISCUSSION TITLE
Roman Numeral Analysis in V4.4

### DISCUSSION THREAD

**Discussion by hugomac65** [musescore/MuseScore#24301] [General]
In the old version, Campania was the default RNA font. It has been switched to Arial in 4.4 so the functionality is compromised. Can Campania be reinstated?

**Comment by MarcSabatella** [comment]
It was omitted due to an oversight and is already fixed for the next update, which will probably happen next week.  Meanwhile, you can install the font yourself from here - https://github.com/MarcSabatella/Campania
